joint_commands_topic: /topic_based_joint_commands
joint_states_topic: /topic_based_joint_states
joint_names: [X_joint, Y_joint, Z_joint, A_joint, B_joint, C_joint]
publish_rate_hz: 50.0
